usaspending_api/matviews/models/__init__.py

```diff
@@ -0,0 +1,46 @@
+from usaspending_api.matviews.models.mv_award_summary import AwardSummaryMatview
```

Comment:
Is a generic "matviews" app better than defining matviews where they are primarily used?

For example, the `mv_*_award_search` materialized views are primarily for the search app.

And `vw_award_search` isn't a materialized view, but a traditional view.